convert.py

```
@@ -1,54 +1,133 @@
-        ['npx', '-y', 'terser', '--compress', '--mangle'],
-        input=js_code,
-        capture_output=True,
-        text=True,
-    )
-    if result.returncode != 0:
-        raise RuntimeError(
-            f"terser minification failed (exit code {result.returncode}):\n"
-            f"{result.stderr}"
+    try:
+        result = subprocess.run(
+            ['npx', '-y', 'terser', '--compress', '--mangle'],
```

Ruff (0.15.18)

[error] Starting a process with a partial executable path

(S607)